### PR TITLE
WIP: Adding Likelihood Q Ratio Test (LQRT)

### PR DESCRIPTION
**Pull request by adam2392**
Addresses: #83 

**Description:** This is a PR that adds the LQRT robust statistical test, which is proven in [1] to be more robust in terms of power for fixed alpha level compared to both the t-test and the Wilcoxon rank-sum test. The python package at: https://github.com/alyakin314/lqrt implements this in Python in both 1-sample and 2-sample case with the possibility of unequal variances to be as general as possible.

Currently work in progress.

References:
[1]: Qin, Yichen & E. Priebe, Carey. (2017). Robust Hypothesis Testing via Lq-Likelihood. Statistica Sinica. 27. 10.5705/ss.202015.0441.

### CODE REVIEW

File: dabest/_classes.py [hunk 1]
```diff
@@ -658,6 +659,16 @@ def __init__(self, control, test, effect_size,
                 # in terms of rank (eg. all zeros.)
                 pass
 
+            # Likelihood Q-Ratio test:
+            # Assumes a gross-error model of distributions
+            try:
+                lqrt_ratio = lqrt.lqrtest_rel(control, test)
```
Comment (josesho) [inline dabest/_classes.py]: Hi @adam2392 

The PR failed automatic Travis CI checks; it seems you used  `lqrt.lqrtest_rel()` for both paired and unpaired cases. This creates an error when the control and test arrays have different lengths.

Could you include a check where `lqrt.lqrtest_ind()` is used if `is_paired` is False?

Comment (adam2392) [reply dabest/_classes.py]: @josesho done!

**Comment by adam2392** [comment]
Let me know what else would be necessary at this point to move this into the release stage. I would love to get this working to use for myself :p.

TODO:?
- [ ]

**Comment by josesho** [comment]
Hi @adam2392 ,

Tests are passing, great! Here's what's left to do.

TODO:
- [ ] Update release-notes.rst
- [ ] Update version number (0.2.8) in setup.py and __init__.py
- [ ] Add a test in `test_01_effsizes_pvals.py` to check that LqRT results (for simple 2-group paired and unpaired cases)

**Pull request merged**